Sources/App/Setup/routes.swift

```diff
@@ -0,0 +1,15 @@
+import Vapor
+
+/// Register your application's routes here.
+public func routes(_ router: Router) throws {
+    // Basic "Hello, world!" example
+    router.get("hello") { _ in
```

Comment:
Similar to the comment I left on the controller above, the `routes` function should be reserved for registering `RouteCollection`s and should not house any actual routes